### PR TITLE
Fixed a few spelling errors in Readme.md

### PR DESCRIPTION
**Pull request by Rishi-coder11**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ Litmus takes a cloud-native approach to create, manage and monitor chaos. Chaos
 - **ChaosExperiment**: A resource to group the configuration parameters of a chaos experiment. ChaosExperiment CRs are created by the operator when experiments are invoked by ChaosEngine.
 - **ChaosResult**: A resource to hold the results of a chaos-experiment. The Chaos-exporter reads the results and exports the metrics into a configured Prometheus server.
 
-Chaos experiments are hosted on <a href="https://hub.litmuschaos.io" target="_blank">hub.litmuschaos.io</a>. It is a central hub where the application developers or vendors share their chaos experiments so that their users can use them to increase the resilience of the applications in production.
+Chaos experiments are hosted on <a href="https://
+
+" target="_blank">hub.litmus chaos.io</a>. It is a central hub where the application developers or vendors share their chaos experiments so that their users can use them to increase the resilience of the applications in production.
```
Comment (S-ayanide) [inline README.md:30]: There is an empty new line which breaks the link in the anchor tag, besides this change doesn't really modify/update any content here.

File: litmus-portal/frontend/README.md [hunk 2]
```diff
@@ -1,46 +1,21 @@
...
-[![YouTube Channel](https://img.shields.io/badge/YouTube-Subscribe-red)](https://www.youtube.com/channel/UCa57PMqmz_j0wnteRa9nCaw)
-<br><br>
-
-<img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fpmcdeadline2.files.wordpress.com%2F2016%2F09%2Ftwitter-logo.jpg&f=1&nofb=1" alt="LitmusChaos Twitter" width="12"> (https://twitter.com/LitmusChaos/) **Follow us on Twitter!** [**@LitmusChaos**](https://twitter.com/LitmusChaos/)
-
-<a href="https://litmuschaos.io"><img src="https://i.ibb.co/QKSMq9g/Screenshot-from-2020-09-14-18-54-05.png" align="right" width="250px" /></a>
-
-## **Learn about Litmus** 😮😎
-
+Learn about Litmus 😮😎
```
Comment (S-ayanide) [inline litmus-portal/frontend/README.md:1]: We'd prefer having the previous readme content along with styling here

File: litmus-portal/frontend/README.md [hunk 3]
```diff
@@ -1,46 +1,21 @@
...
+Table of Contents
+Project Structure Practices
+Available Scripts
+Tech Stack
+Frontend Contribution Guidelines
+Error Handling Practices
+Testing And Overall Quality Practices
+Project Structure Practices
+Coding Style Guide
+User Guide for Litmus Portal
```
Comment (S-ayanide) [inline litmus-portal/frontend/README.md:14]: This removes the Markdown styling and links based on table content, thus we would prefer having the previous one

File: litmus-portal/frontend/README.md [hunk 4]
```diff
@@ -75,115 +50,68 @@ frontend
...
 ├── 📃tsconfig.prod.json
 ├── 📃tsconfig.test.json
-```
 
-<br>
-
-### `2. Available Scripts`
-
-<br>
+2. Available Scripts
```
Comment (S-ayanide) [inline litmus-portal/frontend/README.md:54]: This is supposed to be a bold list item which is linked with the Table of Contents.

File: litmus-portal/frontend/README.md [hunk 5]
```diff
@@ -75,115 +50,68 @@ frontend
...
-It correctly bundles React in production mode and optimizes the build for the best performance.
-`Prettier` for formatting and `linters` for catching bugs! 🐛
-<br><br>
+Formats your document according to the prettier guidelines and then does a eslint --fix Builds the app for production to the build folder.
+It correctly bundles React in production mode and optimizes the build for the best performance. Prettier for formatting and linters for catching bugs! 🐛
 
-### `3. Tech Stack`
 
-<br>
+3. Tech Stack
```
Comment (S-ayanide) [inline litmus-portal/frontend/README.md:89]: This is also supposed to be a bold list item which is linked with the Table of Contents.

File: litmus-portal/frontend/README.md [hunk 6]
```diff
@@ -75,115 +50,68 @@ frontend
...
-<br><br>
+        
 
-### `4. Frontend Contribution Guidelines`
 
-<br>
+4. Frontend Contribution Guidelines
```
Comment (S-ayanide) [inline litmus-portal/frontend/README.md:94]: This is also supposed to be a bold list item which is linked with the Table of Contents.

File: litmus-portal/frontend/README.md [hunk 7]
```diff
@@ -75,115 +50,68 @@ frontend
...
 
-We at Litmus always welcome a good frontend developer with open arms.<br>
+We at Litmus always welcome a good frontend developer with open arms.
 Read our Contribution guidelines to get started quickly with our project 🗒️
-<br><br>
 
-### `5. Error Handling Practices`
 
-<br>
+5. Error Handling Practices
```
Comment (S-ayanide) [inline litmus-portal/frontend/README.md:100]: This is also supposed to be a bold list item which is linked with the Table of Contents. Similarly for the rest of the items as well.

File: litmus-portal/frontend/README.md [hunk 8]
```diff
@@ -75,115 +50,68 @@ frontend
...
-<br><br>
+        
 
-### `4. Frontend Contribution Guidelines`
 
-<br>
+4. Frontend Contribution Guidelines
 
-We at Litmus always welcome a good frontend developer with open arms.<br>
+We at Litmus always welcome a good frontend developer with open arms.
```
Comment (S-ayanide) [inline litmus-portal/frontend/README.md:96]: We'd much prefer having a breakpoint there to make it space out.

File: litmus-portal/frontend/README.md [hunk 9]
```diff
@@ -75,115 +50,68 @@ frontend
...
 
-<img src = 'https://image.flaticon.com/icons/svg/919/919853.svg' width='30'/> <img src = 'https://external-content.duckduckgo.com/iu/?u=http%3A%2F%2Fwww.codeproject.com%2FKB%2FHTML%2F843860%2FTypeScript.png&f=1&nofb=1' width='30'/> <img src = 'https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fcdn.volaresystems.com%2FImages%2FPosts%2F2019%2F12%2Feslint_logo_text.png&f=1&nofb=1' width='30'/> <img src = 'https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ffaridoon.com%2Fwp-content%2Fuploads%2F2019%2F08%2FMaterialUILogo.png&f=1&nofb=1' width='30'/> <img src = 'https://image.flaticon.com/icons/svg/919/919851.svg' width='30'/> <img src = 'https://image.flaticon.com/icons/svg/919/919856.svg' width='30'/> <img src = 'https://upload-icon.s3.us-east-2.amazonaws.com/uploads/icons/png/3556671901536211770-512.png' width='30'/> <img src = 'https://upload.wikimedia.org/wikipedia/commons/thumb/1/17/GraphQL_Logo.svg/1200px-GraphQL_Logo.svg.png' width='30'/> <img src = 'https://d2eip9sf3oo6c2.cloudfront.net/tags/images/000/001/216/thumb/apollo-seeklogo.com_%281%29.png' width='30'/><img src = 'https://cdn.iconscout.com/icon/free/png-512/redux-283024.png' width='30'>
-<br><br>
+
```
Comment (S-ayanide) [inline litmus-portal/frontend/README.md:91]: This change deletes all the Tech Stack images which were previously present, thus this can't be accepted.

**Review by S-ayanide — CHANGES_REQUESTED** [review]
Thanks for sending a PR to Litmus  🎉 

This PR modifies the content of the readme and rather deletes most of it instead of adding content to it. We're happy with the previous Readme in terms of styling thus we are looking forward to having content changes and not style changes in Markdown. 

If you could add more content to the ones already present based on Configuration, Guidelines, Scripts, that would be great.

**Comment by arkajyotiMukherjee** [comment]
If you feel this PR shouldn't have been closed you can provide your justification and reason for it and it might be reopened.
Thanks for investing your time in Litmuschaos.

**Pull request closed without merging**